webapp/tests/forms/flows/test_lotse_flow.py

```diff
@@ -1425,7 +1431,7 @@ def test_if_contains_not_all_mandatory_fields_but_all_confirmations_then_raise_i
         expected_missing_fields = ['steuernummer_exists', 'bundesland', 'bufa_nr', 'request_new_tax_number', 'familienstand', 'person_a_dob',
                                    'person_a_last_name', 'person_a_first_name', 'person_a_religion', 'person_a_street',
                                    'person_a_street_number', 'person_a_plz', 'person_a_town', 'person_a_blind',
-                                   'person_a_gehbeh', 'person_a_has_disability', 'iban', 'is_user_account_holder', ]
+                                   'person_a_gehbeh', 'person_a_has_disability', 'person_a_requests_pauschbetrag', 'iban', 'is_user_account_holder', ]
```

Comment:
Why should the field be missing? It really shouldn't be mandatory if `person_a_has_disability` is not given?

Reply:
Ur right :grin: